deepinv/optim/data_fidelity.py

```diff
@@ -278,6 +282,234 @@ def prox(self, x, y, physics, *args, gamma=1.0, **kwargs):
         return physics.prox_l2(x, y, self.norm * gamma)
 
 
+class ItohFidelity(L2):
+    r"""
+    Itoh data-fidelity term for spatial unwrapping problems.
+
+    This class implements a data-fidelity term based on the normalized :math:`\ell_2` norm, but applied to the spatial finite differences of the variable and the wrapped differences of the data.
```

Comment:
```suggestion
    This class implements a data-fidelity term based on the :math:`\ell_2` norm, but applied to the spatial finite differences of the variable and the wrapped differences of the data.
```

not sure why 'normalized'?

Reply:
Sorry, my bad, it was a typo.